manzif/Banka
# #165610679 Ft testing 165610679

### What does this PR do?
Add Tests to the codes

### Description of Task to be completed?
create a database for tests
create a folder for test
Add testing for all the code

### How should this be manually tested?
`npm run server: start` to the run the app corresponding to this PR and  you run this `npm run test`
 You will see the testing of all the codes

### What are the relevant pivotal tracker stories?
[#165610679](https://www.pivotaltracker.com/story/show/165610679)
